### ISSUE TITLE
Android : How to remove underline coming in PaymentCardTextField ?

### ISSUE THREAD

**Issue by ayushnawani** [tipsi/tipsi-stripe#146]
Any idea how can i remove blue underline in PaymentCardTextField in android ?
I have tried using textinput props "underlineColorAndroid", but it was not working.

**Comment by isnifer** [comment]
@ayushnawani as I can see in our project, please try to apply `borderWidth: 0`

**Comment by ayushnawani** [comment]
Still its showing blue underline after borderWidth:0, it is anyway related to [this issue ?](https://github.com/facebook/react-native/issues/5424)

**Comment by isnifer** [comment]
@ayushnawani not sure. Let's try to synchronize, show me your code for `PaymentCardTextField`.

**Comment by isnifer** [comment]
@ayushnawani this is our view:
![image](https://user-images.githubusercontent.com/1788245/31898091-e6aa1402-b820-11e7-9d30-ed174e0823f1.png)

And this is our code:
```js
<PaymentCardTextField
  ref={this.paymentCardFieldRef}
  style={styles.cardField}
  placeholderColor="#d7d5d8"
  onParamsChange={this.handleFieldParamsChange}
/>
```
Also there is a style for that:
```js
cardField: {
  width: window.width - 13 - 20,
  height: 44,
  borderWidth: 0,
  marginLeft: Platform.select({ ios: -13, android: -18 }),
},
```

**Comment by ayushnawani** [comment]
This is the code. 
```
<View style={{
        borderColor: '#27dc95',
        borderWidth: 1,
        borderRadius: 5,
        width: width-85,
        marginLeft: 10,
        height: height/15
}}>
    <PaymentCardTextField
        accessible={false}
        style={{
            width: width-80,
            borderWidth: 0,
            height: 35
        }}
        onParamsChange={this.handleFieldParamsChange}
        {...testID('cardTextField')}
    />
</View>
```


And  View 

<img width="371" alt="cardscreen" src="https://user-images.githubusercontent.com/15374835/31925446-5762b232-b8a6-11e7-91ed-4ab5cc3e6b89.png">

**Comment by anjali14garg** [comment]
I am also facing this issue . And update regarding this issue

**Comment by paynd** [comment]
@isnifer @ayushnawani 
Fixed in https://github.com/tipsi/tipsi-stripe/pull/148

**Comment by isnifer** [comment]
@ayushnawani @anjali14garg fixed in 3.9.1. Thank you for your issue!

**Comment by ayushnawani** [comment]
Thank you for the fix @paynd and @isnifer
